From 3630a8b1e1cd99862867fe72ffa1ff51e4d9c09f Mon Sep 17 00:00:00 2001
From: iglocska <andras.iklody@gmail.com>
Date: Fri, 10 Mar 2017 09:17:06 +0100
Subject: [PATCH] fix: Tightened sanitisation in some view elements - on the
 index filter tool - organisation landing page

as reported by Tien Phan and David Maciejak of Fortinet's FortiGuard Labs.
---
 app/View/Organisations/ajax/landingpage.ctp     | 2 +-
 app/webroot/js/{misp2.4.67.js => misp2.4.68.js} | 0
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename app/webroot/js/{misp2.4.67.js => misp2.4.68.js} (100%)

diff --git a/app/View/Organisations/ajax/landingpage.ctp b/app/View/Organisations/ajax/landingpage.ctp
index 0d21e560d7..4e556581df 100644
--- a/app/View/Organisations/ajax/landingpage.ctp
+++ b/app/View/Organisations/ajax/landingpage.ctp
@@ -1,5 +1,5 @@
 <div>
-<h3>Landing page for <?php echo $org;?></h3>
+<h3>Landing page for <?php echo h($org);?></h3>
 <div>
 <?php echo h($landingPage);?>
 </div>
diff --git a/app/webroot/js/misp2.4.67.js b/app/webroot/js/misp2.4.68.js
similarity index 100%
rename from app/webroot/js/misp2.4.67.js
rename to app/webroot/js/misp2.4.68.js
